lib/theme/widgets/hs_warning_bottom_sheet.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class HsWarningBottomSheetView extends StatelessWidget {
```

Comment:
The current implementation only supports two buttons: primary and secondary. It should support tertiary and text buttons as well

Reply:
already added more buttons